typescript/package.json

```diff
@@ -9,6 +9,13 @@
   "workspaces": [
     "./common/*"
   ],
+  "dependencies": {
+    "@bytecodealliance/jco": "=1.4.4",
+    "@bytecodealliance/componentize-js": "=0.10.5"
+  },
+  "bin": {
+    "jco": "node_modules/@bytecodealliance/jco/src/jco.js"
```

Comment:
FYI: We do this so that our root package may constitute an effective "re-export" of `jco`'s `bin`. This is a sort of hack to get around `nix`'s impedance mismatch with global environment dependencies.